src/query/block/empty.go

```diff
@@ -113,3 +113,17 @@ func (it *ucEmptySeriesIter) SeriesCount() int              { return 0 }
 func (it *ucEmptySeriesIter) SeriesMeta() []SeriesMeta      { return []SeriesMeta{} }
 func (it *ucEmptySeriesIter) Next() bool                    { return false }
 func (it *ucEmptySeriesIter) Current() UnconsolidatedSeries { return UnconsolidatedSeries{} }
+
+func (b *ucEmptyBlock) MultiSeriesIter(
+	concurrency int,
+) ([]UnconsolidatedSeriesIterBatch, error) {
+	batch := make([]UnconsolidatedSeriesIterBatch, concurrency)
```

Comment:
Would it make sense to use append for consistency?

Reply:
For cases like this where we're initializing the entire slice to a default value I think this reads a little clearer than doing:

```
batch := make([]UnconsolidatedSeriesIterBatch, 0, concurrency)
for i := 0; i < concurrency; i++ {
```

What do you think?